src/components/header/index.tsx

```diff
@@ -19,11 +19,11 @@ export default function Header() {
 
   const handleScroll = useCallback(() => {
     setTimeout(() => {
-      if (window.scrollY < 1) {
+      if (window.scrollY < 1 || !routerIsMain) {
         setShrinkHeader(false);
       } else setShrinkHeader(true);
     }, 200);
-  }, []);
+  }, [router]);
```

Comment:
should routerIsMain be here? What are you attempting with this component?

Reply:
I wanted to disable the shrinkHeader animation when it's not on '/' or  '/learn'

Reply:
I wanted to disable the shrinkHeader animation when it's not on '/' or '/learn'